src/main/java/org/isf/accounting/model/Bill.java

```diff
@@ -74,18 +74,18 @@ public class Bill extends Auditable<String> implements Comparable<Bill>
 	private int id;
 	
 	@NotNull
-	@Column(name="BLL_DATE")
-	private GregorianCalendar date;
+	@Column(name="BLL_DATE")		// SQL type: datetime
+	private LocalDateTime date;
 
 	@NotNull
-	@Column(name="BLL_UPDATE")
-	private GregorianCalendar update;
+	@Column(name="BLL_UPDATE")		// SQL type: datetime
+	private LocalDateTime update;
 
 	@NotNull
 	@Column(name="BLL_IS_LST")
 	private boolean isList;
 	
-	@ManyToOne
+	@ManyToOne(cascade = {CascadeType.MERGE})
```

Comment:
@pviotti we are going to remove this cascade ([OP-990](https://openhospital.atlassian.net/browse/OP-990)). Did you remember why it was introduced?

Reply:
:thinking: most likely it was introduced to avoid compilation errors with the mysql-related version bumps of this PR - I can't remember any other possible reason.
It changes the way Patient and Bill [are persisted when they're handled together](https://www.baeldung.com/jpa-cascade-types#3-cascadetypemerge), and it kinda make sense - but if necessary you can try to remove it.

Reply:
Yes we are removing them, because the child entity is not created together with the parent, they have independent life (see OP-990). Thanks!